s3/container.go

```diff
@@ -78,7 +78,7 @@ func (c *container) Items(prefix, startAfter string, count int) ([]stow.Item, st
 
 	// Create a marker and determine if the list of items to retrieve is complete.
 	// If not, the last file is the input to the value of after which item to start
-	startAfter := ""
+	startAfter = ""
```

Comment:
I'd say we create a new variable here... doens't make much sense to reuse it, sorry for our confusion and thank you for the contribution :-D

Reply:
and just do `var foo string`